scripts/cnode-helper-scripts/cnode.sh

```diff
@@ -25,27 +25,33 @@ if [[ -S "${CARDANO_NODE_SOCKET_PATH}" ]]; then
   fi
 fi
 
+[[ -z ${CPU_CORES} ]] && CPU_CORES=2
+
 [[ ! -d "${LOG_DIR}/archive" ]] && mkdir -p "${LOG_DIR}/archive"
 
 [[ $(find "${LOG_DIR}"/*.json 2>/dev/null | wc -l) -gt 0 ]] && mv "${LOG_DIR}"/*.json "${LOG_DIR}"/archive/
 
+host_addr=()
+[[ ${IP_VERSION} = "4" || ${IP_VERSION} = "mix" ]] && host_addr+=("--host-addr" "0.0.0.0")
+[[ ${IP_VERSION} = "6" || ${IP_VERSION} = "mix" ]] && host_addr+=("--host-ipv6-addr" "::/0")
+
 if [[ -f "${POOL_DIR}/${POOL_OPCERT_FILENAME}" && -f "${POOL_DIR}/${POOL_VRF_SK_FILENAME}" && -f "${POOL_DIR}/${POOL_HOTKEY_SK_FILENAME}" ]]; then
-  cardano-node run \
+  cardano-node +RTS -N${CPU_CORES} -RTS run \
```

Comment:
Unless a custom CPU_CORES is specified, it would be preferable to have this empty. I believe we should not force users up/down to 2 CPU defaults unless we really need to - also, keeping in line wider documentation.

Reply:
Fixed in d0712a3455631a5db8c2fb7f6ccf4ff4ae99cdd4